### PR TITLE
Simplified api - bbCreateStore()

### PR DESCRIPTION
**Pull request by adamterlson**
TODO: Tests :D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ Writen by Vitor Balocco ([@vitorbal](https://github.com/vitorbal)) and Adam Terl
 ## Usage
 
 ```javascript
-import { createBBStore, combineBBEntities } from 'simple-backbone-redux';
+import { bbCreateStore, bbCombineEntities } from 'simple-backbone-redux';
 
-const bbStore = createBBStore(combineBBEntities({
+const bbStore = bbCreateStore(bbCombineEntities({
```
Comment (vitorbal) [inline README.md]: we don't need to call `bbCombineEntities` here anymore, I think? doesn't `bbCreateStore` do that for us now?

Comment (adamterlson) [inline README.md]: Yes.  This README update was just a find-replace for the old method changing names.  Really it's still a big TODO to update the main README with all the appropriate usage.

**Comment by vitorbal** [comment]
Awesome. 🚢  it. Just a tiny thing in the README. I can work on the tests tonight probably.

**Comment by adamterlson** [comment]
@vitorbal Great!  Just feel free to merge it if you want to create a separate PR for the tests, or you can of course just update this branch, whichever you prefer.

**Comment by vitorbal** [comment]
Loving the new README :tada:

**Comment by vitorbal** [comment]
LGTM

**Pull request merged**